ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/castor/algorithm/AutomatonSimulator.java

```diff
@@ -391,11 +389,6 @@ static boolean checkAlignedRangeActions(State automatonState, List<RangeAction<?
             BUSINESS_WARNS.warn("Range action group {} contains range actions of different types; they are not simulated", alignedRa.get(0).getGroupId().orElseThrow());
             return false;
         }
-        // Ignore aligned range actions when one element of the group has a different usage method than the others
-        if (alignedRa.stream().map(rangeAction -> rangeAction.getUsageMethod(automatonState)).distinct().count() > 1) {
-            BUSINESS_WARNS.warn("Range action group {} contains range actions with different usage methods; they are not simulated", alignedRa.get(0).getGroupId().orElseThrow());
-            return false;
-        }
```

Comment:
why did you remove this?

Reply:
Before using this method (only once in the code) we select only available ra (meaning with usageMethod FORCED and with flow constraint for ofc usageRules). Before we used to have FORCED and TO_BE_EVALUATED ones but not now so this check seems useless to me.
``` java
// -- First get forced range actions
Set<RangeAction<?>> availableRangeActions = crac.getRangeActions().stream()
  .filter(ra -> RaoUtil.isRemedialActionAvailable(ra, automatonState, rangeActionSensitivity, crac.getFlowCnecs(), network, raoParameters))
  .collect(Collectors.toSet());
```